metadata-io/src/main/java/com/linkedin/metadata/entity/cassandra/CassandraAspectDao.java

```diff
@@ -431,6 +431,13 @@ public int deleteUrn(@Nullable TransactionContext txContext, @Nonnull final Stri
     return rs.getExecutionInfo().getErrors().size() == 0 ? -1 : 0;
   }
 
+  @Override
+  public int deleteUrn(
+      @Nullable TransactionContext txContext, @Nonnull String urn, @Nonnull String keyAspect) {
+    throw new UnsupportedOperationException(
```

Comment:
This should be fairly easy to convert? Likely better to just route it in to the standard deleteUrn function in case its use gets expanded

Reply:
https://github.com/datahub-project/datahub/pull/14824#discussion_r2392186795

^ as long as we have a ticket for this, this can be considered resolved